### DISCUSSION TITLE
There is a docker-compose example for Redash with elasticsearch and kibana ?

### DISCUSSION THREAD

**Discussion by survivant** [getredash/redash#6028] [Q&A]
I'm trying to find a working sample for Redash and elasticsearch integration.  I wasn't able to find any :(

I was able to get a docker-compose file to start elastic search and kibana (without user/password) and I found a docker-compose for redash too.  When I try to add a elasticsearch connection.  I always got a Connection Test Failed.  I left the basic authentication blank because there wasn't any authentication in ES.  I use the Elastic search connection not AWS.  

I'm running that on my local computer.

I try also a to add the authentification for ES, but it still didn't work in Redash.  (I was able to log into ES at localhost:9200 with the right credentials).

**Comment by mariovonleitmeritz** [comment]
[compose.txt](https://github.com/getredash/redash/files/15127111/compose.txt)

In the Elasticsearch datasource set Base URL to http://elastic:9200 user and password leave blank, 
save datasource - this disable the "Test connection" button. Open the saved datasource and press Test connection button
